### PR TITLE
Add problem for simulation and solution

### PR DESCRIPTION
**Pull request by chuminjae**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [ ✓] I have tested my code.
- [ ✓] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ✓] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ✓] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/bronze/cf-1914C.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+id: cf-1914C
+source: CF 1914C
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-1914C.mdx:3]:
```suggestion
source: CF
```

**Comment by SansPapyrus683** [comment]
ngl feel like there's enough simulation problems alr

**Review by envyaims — COMMENTED** [review]
its whatever, quite trivial imo
also isnt really simulation? what are you simulating?

**Comment by chuminjae** [comment]
> its whatever, quite trivial imo also isnt really simulation? what are you simulating?

more similar with implementation.

**Review by envyaims — COMMENTED** [review]
maybe move it to additional problems?

File: solutions/bronze/cf-1914C.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+id: cf-1914C
+source: CF
+title: Quests
+author: Chu Minjae
+---
+
+## Implementation
+
+We can find the maximum experience point available by comparing the largest available experience point gained by completing one more quest or not.
```
Comment (ryanchou-dev) [inline solutions/bronze/cf-1914C.mdx:10]: move above implementation header

File: solutions/bronze/cf-1914C.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+int t, n, k, h;
+int ans;
+int tmp;
+int limit;
```
Comment (ryanchou-dev) [inline solutions/bronze/cf-1914C.mdx:23]: add extra lines above and below this

File: solutions/bronze/cf-1914C.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+		tmp = a[0];
+		for (int z = 1; z < limit; z++) {
+			mx = max(mx, b[z]);
+			// compare experience point of completing one more quest or not
+			if (ans < tmp + a[z] + mx * (k - z - 1)) {
+				ans = tmp + a[z] + mx * (k - z - 1);
+			}
+			tmp += a[z];
+		}
+		cout << ans << "\n";
```
Comment (ryanchou-dev) [inline solutions/bronze/cf-1914C.mdx:45]: would be cool if you could space out some of these lines as well

**Review by ryanchou-dev — COMMENTED** [review]
not sure where this fits in with the bronze module, i could see an argument being made for the additional problems section though

File: solutions/bronze/cf-1914C.mdx (hunk 3 above)
Comment (SansPapyrus683) [reply solutions/bronze/cf-1914C.mdx:23]: these variables should be declared where they're used
just in general, you shouldn't declare all the variables at the start...

**Comment by chuminjae** [comment]
Where should I place this?

**Comment by SansPapyrus683** [comment]
additional problems

**Comment by chuminjae** [comment]
Should I also move the solution?

**Comment by envyaims** [comment]
> Should I also move the solution?

you didn't move it to the right place. There should be an additional problems mdx in the bronze content directory

**Comment by chuminjae** [comment]
I don't see the additional problems in the bronze content directory.

**Comment by ryanchou-dev** [comment]
`usaco-guide/content/2_Bronze/Conclusion.problems.json`

File: solutions/bronze/cf-1914C.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+		vector<int> b(n);
+
+		// farthest location of available quest
+		int limit = min(n, k);
+		for (int x = 0; x < n; x++) { cin >> a[x]; }
+		for (int y = 0; y < n; y++) { cin >> b[y]; }
+
+		int mx = b[0];
+		int ans = a[0] + mx * (k - 1);
+		int tmp = a[0];
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-1914C.mdx:38]: better name than tmp

File: solutions/bronze/cf-1914C.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+		int mx = b[0];
+<<<<<<< HEAD
+		// ans here represents the case only completing the first
+		int ans = a[0] + mx * (k - 1);
+		// ans1 is used to calculate the case completing other quests
+		int ans1 = a[0];
+=======
+		int ans = a[0] + mx * (k - 1);
+		int tmp = a[0];
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-1914C.mdx:44]: bruh

File: solutions/bronze/cf-1914C.mdx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+		vector<int> a(n);
+		vector<int> b(n);
+		for (int x = 0; x < n; x++) { cin >> a[x]; }
+		for (int y = 0; y < n; y++) { cin >> b[y]; }
+
+		int mx = b[0];
+		// ans here represents the case only completing the first
+		int ans = a[0] + mx * (k - 1);
+		// anotherQuest is used to calculate the case completing other quests
+		int anotherQuest = a[0];
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-1914C.mdx:40]: use snake_case

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
awesome

**Pull request merged**